services/localfile/localfile.proto

```diff
@@ -68,6 +68,12 @@ service LocalFile {
 
   // Mkdir create a new directory.
   rpc Mkdir(MkdirRequest) returns (google.protobuf.Empty) {}
+
+  // Get property from a file of specified type by provided key
```

Comment:
nit: property doesn't seem to be the right word. IMO property sounds like file metadata

Reply:
Updated